src/main/java/uk/gov/hmcts/befta/util/DynamicValueInjector.java

```diff
@@ -111,6 +113,18 @@ private Object processDynamicValuesIn(String input) {
                 String formulaPart = input.substring(pos, closingAt + 1);
                 partValue = calculateFormulaFromContext(scenarioContext, formulaPart);
                 jumpTo = closingAt + 1;
+
+                // special case when complete input is a dynamic formula
+                // : so check if it returned a complex object to return
+                if ((pos == 0 && jumpTo >= input.length())
+                        && (partValue == null
+                        || partValue instanceof Map<?, ?>
+                        || partValue instanceof Object[]
+                        || partValue instanceof Iterable)
+                ) {
```

Comment:
Let's remove this condition.

Reply:
Done: and local tests against data-store FTA remain green. 👍